src/Application/Actions/Hooks/Stripe.php

```diff
@@ -60,13 +60,18 @@ protected function prepareEvent(
         }
 
         if (!$this->event->livemode && getenv('APP_ENV') === 'production') {
+            $logMessage = sprintf('Skipping non-live %s webhook in Production', $this->event->type);
             /**
              * This is normal for Connect events so just `info()` log it in that case.
              * "...both live and test webhooks will be sent to your production webhook URLs."
              * @link https://stripe.com/docs/connect/webhooks
              */
-            $method = $connect ? 'info' : 'warning';
-            $this->logger->$method(sprintf('Skipping non-live %s webhook in Production', $this->event->type));
+            if ($connect) {
+                $this->logger->info($logMessage);
```

Comment:
I guess we could also call `log()` with the level as an arg? More concise but maybe not clearer, not a blocker.

Reply:
yeah I was going to do that but the first argument to `log` on the psr interface is `mixed` so I didn't like it.